pkg/accounting/rollup/rollup_test.go

```diff
@@ -8,79 +8,105 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
-	"go.uber.org/zap"
 
 	"storj.io/storj/internal/testcontext"
-	"storj.io/storj/internal/teststorj"
-	"storj.io/storj/pkg/accounting/rollup"
+	"storj.io/storj/internal/testplanet"
 	"storj.io/storj/pkg/storj"
-	"storj.io/storj/satellite"
-	"storj.io/storj/satellite/satellitedb"
 )
 
-func TestQueryOneDay(t *testing.T) {
-	// TODO: use testplanet
-
-	ctx, r, db, nodeData, cleanup := createRollup(t)
-	defer cleanup()
+func TestQuery(t *testing.T) {
+	tests := []struct {
+		days   int
+		atRest float64
+		bw     []int64
+		nodes  int
+	}{
+		{
+			days:   1,
+			atRest: float64(5000),
+			bw:     []int64{1000, 2000, 3000, 4000},
+			nodes:  4,
+		},
+		{
+			days:   2,
+			atRest: float64(10000),
+			bw:     []int64{2000, 4000, 6000, 8000},
+			nodes:  6,
+		},
+		{
+			days:   5,
+			atRest: float64(20000),
+			bw:     []int64{4000, 8000, 12000, 16000},
+			nodes:  8,
+		},
+	}
 
-	now := time.Now().UTC()
-	later := now.Add(time.Hour * 24)
+	for _, tt := range tests {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
 
-	err := db.Accounting().SaveAtRestRaw(ctx, now, nodeData)
-	assert.NoError(t, err)
+		planet, err := testplanet.New(t, 1, tt.nodes, 0)
+		assert.NoError(t, err)
+		defer ctx.Check(planet.Shutdown)
 
-	// test should return error because we delete latest day's rollup
-	err = r.Query(ctx)
-	assert.NoError(t, err)
+		planet.Start(ctx)
```

Comment:
Btw. we now also have `testplanet.Run` which tests against both sqlite and postgres.

Reply:
this really would be nice to have